caffe2/onnx/helper.h

```diff
@@ -12,6 +12,7 @@ namespace onnx {
 
 using ::ONNX_NAMESPACE::AttributeProto;
 using ::ONNX_NAMESPACE::NodeProto;
+using ::ONNX_NAMESPACE::TensorProto;
```

Comment:
Using TensorProto here is not recommended, especially, it's a header file.
And another reason is that we have both caffe2::TensorProto and onnx::TensorProto (caffe2 does not have NodeProto and AttributeProto). I would suggest explicitly use onnx::TensorProto in the following code.

Reply:
Yeah that's a good idea.

Reply:
wait, you mean ONNX_NAMESPACE::TensorProto bot onnx::TensorProto, right?